hypothesis-python/src/hypothesis/internal/conjecture/data.py

```diff
@@ -1450,6 +1450,9 @@ def __init__(
         self.arg_slices: Set[Tuple[int, int]] = set()
         self.slice_comments: Dict[Tuple[int, int], str] = {}
         self._observability_args: Dict[str, Any] = {}
+        self._observability_predicates: defaultdict = defaultdict(
+            lambda: {"sat": 0, "unsat": 0}
+        )
```

Comment:
nit: since we're exposing this in observability reports, the full names of "satisfied" and "unsatisfied" make more sense to me from the perspective of a library parsing our observability spec. Bringing this up because it's easy to change now, as opposed to when the format has stabilized a bit (and potentially has stable consumers).